apps/new/widget/lib/projects.jsx

```diff
@@ -100,7 +100,7 @@ const getProjectMeta = (id) => {
 };
 
 const getProjectIdFromPath = (id) => {
-  return id.split("/")[2] ?? null;
+  return (id ?? "").split("/")[2] ?? null;
```

Comment:
Why did this change need to go in?

Reply:
if we open the editor without any id (i.e for creating a project), it will break, since `null/undefined.split()` is not valid